packages/vm/src/testing.rs

```diff
@@ -83,7 +83,7 @@ pub fn handle<
     msg: T,
 ) -> HandleResult<U> {
     match to_vec(&msg) {
-        Err(e) => Err(e.into()),
+        Err(e) => Err(e),
```

Comment:
Now that we don't need `e.into()`, this can be simplified:

```rust
let serialized_msg = to_vec(&msg)?;
call_handle(instance, &env, &serialized_msg).unwrap()
```

I think there are a number of other places with similar updates. Passing the same error type through makes things much simpler.

Reply:
Good point. Found a few places in https://github.com/CosmWasm/cosmwasm/pull/305/commits/31f6e7feaddde19cc727b65c1ca8185f287c5067